src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -458,8 +458,9 @@ input.inline {
     width: 60%;
 }
 
-.panel-horizontal {
-    display: inline-block;
+.htCenter {
+    overflow-y: auto;
+    height: 500px;
```

Comment:
These rules are ignored because they are not as specific as the default handsontable rules. Anyway, I find these 2 rules produce weird results. I think this kind of thing should be managed by handsontable.